### ISSUE TITLE
Installer: Works but throws sql exception in installer output.

### ISSUE THREAD

**Issue by kcondon** [IQSS/dataverse#3303]
Attempting to deploy the application.
Command line: /usr/local/glassfish4/bin/asadmin deploy dataverse.war

PER01003: Deployment encountered SQL Exceptions:
    PER01000: Got SQLException executing statement "CREATE INDEX INDEX_DATAVERSE_fk_dataverse_id ON DATAVERSE (fk_dataverse_id)": org.postgresql.util.PSQLException: ERROR: column "fk_dataverse_id" does not exist
Command deploy completed with warnings.

**Comment by pdurbin** [comment]
This error goes back to at least Mar 9, 2016 as seen at https://build.hmdc.harvard.edu:8443/job/phoenix.dataverse.org-deploy-develop/11/console which was a build of https://github.com/IQSS/dataverse/commit/553597070511c4612c1fa394e90a2d7afd9c2ee3 which got tagged in 4.3.

**Comment by djbrooke** [comment]
Removing from the 4.5 milestone as this warning is existing and it still allows the install to continue.

**Comment by pdurbin** [comment]
I can only assume this error is being thrown because of `@Index(columnList="fk_dataverse_id")` at https://github.com/IQSS/dataverse/blob/v4.5.1/src/main/java/edu/harvard/iq/dataverse/Dataverse.java#L48 but I'm not sure if we should remove that index or rename it? I think we last added indexes in #1880 but maybe something was renamed? I'm looking into this because #3515 is related.

**Comment by pdurbin** [comment]
@raprasad says, " Yes, remove it [`@Index(columnList="fk_dataverse_id")`].  This was added in May 21 *2105* -- e.g. it appears that the column was removed in a subsequent release." Seems low-hanging. There's a related issue in #3515 where we should rename `@Index(columnList="md5")` to `@Index(columnList="checksumvalue")` (introduced by #3354 as we worked on 4.6). This seem low-hanging so I'll throw it in the dev column in Waffle.

**Comment by pdurbin** [comment]
In pull request #3520 I removed the warnings by renaming one index from `md5` to `checksumvalue` and by removing the reference to `fk_dataverse_id` entirely.

**Comment by pdurbin** [comment]
> In pull request #3520 I removed the warnings by renaming one index from md5 to checksumvalue and by removing the reference to fk_dataverse_id entirely.

@landreev just reviewed these changes and said they looked ok.

**Comment by pdurbin** [comment]
@kcondon @djbrooke and I just decided to merge pull request #3520.

Things are much better. Prior to the merge you'd see scary messages like this when installing Dataverse:


Command start-domain executed successfully.
PER01003: Deployment encountered SQL Exceptions:
	PER01000: Got SQLException executing statement "CREATE INDEX INDEX_DATAVERSE_fk_dataverse_id ON DATAVERSE (fk_dataverse_id)": org.postgresql.util.PSQLException: ERROR: column "fk_dataverse_id" does not exist
	PER01000: Got SQLException executing statement "CREATE INDEX INDEX_DATAFILE_md5 ON DATAFILE (md5)": org.postgresql.util.PSQLException: ERROR: column "md5" does not exist
**Command deploy completed with warnings.**

That's from https://build.hmdc.harvard.edu:8443/job/phoenix.dataverse.org-deploy-develop/68/console

Now, you see this non-scary message instead:

Command start-domain executed successfully.
Application deployed with name dataverse.
**Command deploy executed successfully.**

That's from https://build.hmdc.harvard.edu:8443/job/phoenix.dataverse.org-deploy-develop/69/console

Closing.
